src/c4/C4_Serial.hh

```diff
@@ -73,6 +73,14 @@ DLL_PUBLIC_c4 int send_udt(const T * /*buffer*/, int /*size*/,
   return C4_SUCCESS;
 }
 
+template <typename TS, typename TR>
+DLL_PUBLIC_c4 int send_receive(TS * /*sendbuf*/, int /*sendcount*/,
+                               int /*destination*/, TR * /*recvbuf*/,
+                               int /*recvcount*/, int /*source*/,
+                               int /*sendtag*/, int /*recvtag*/) {
+  Insist(false, "send_receive is not support for C4_SCALAR builds.");
```

Comment:
We talked about this yesterday. I think we could implement a 'scalar' version of this that simply copies the `sendbuf` into the `recvbuf`.  Theoretically, Draco supports a `C4=SCALAR` (no MPI) mode and I would like to make an attempt keep this mode alive.  Can you make this change, removing the fatal error?